adcm_client/objects.py

```diff
@@ -754,8 +758,9 @@ def try_wait(self, timeout=None):
 
         return status
 
-    def _log_jobs(self, status):
-        for job in self.job_list(status=status):
+    def _log_jobs(self, status=None):
+        job_list = self.job_list(status=status) if status else self.job_list()
+        for job in job_list:
```

Comment:
```suggestion
    def _log_jobs(self, **filters):
      for job in self.job_list(**filters):
```

There is no real needs to pass status when you can pass any of job ojbect attribute. 

But in that case you have to call this function as follows:

```python
            if log_failed and status == "failed":
                self._log_jobs(status="failed")
```

Or even better:
```python
            if log_failed and status == "failed":
                self._log_jobs(status=status)
```

Reply:
Fixed